.github/workflows/build.yml

```diff
@@ -0,0 +1,200 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+# cancel already running builds of the same branch or pull request
+concurrency:
+  group: ci-${{ github.head_ref }} || concat(${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  MATE_PANEL_DEP: 1.27.1
+  CONFIGURE_FLAGS: --enable-compile-warnings=maximum
+  CFLAGS: -g -O2 -Werror=pointer-arith -Werror=implicit-function-declaration
+  JOBS: 2
+  DEBUG: 1
+  # Useful URL: https://github.com/mate-desktop/debian-packages
+  # Useful URL: https://salsa.debian.org/debian-mate-team/mate-panel
+  DEB_LIBRARY_DEPS: |
+    libatk1.0-dev
+    libcairo2-dev
+    libdconf-dev
+    libgirepository1.0-dev
+    libglib2.0-dev
+    libgtk-3-dev
+    libgtk-layer-shell-dev
+    libice-dev
+    libmate-desktop-dev
+    libmate-menu-dev
+    libmateweather-dev
+    libpango1.0-dev
+    libsm-dev
+    libsoup2.4-dev
+    libwnck-3-dev
+    libx11-dev
+    libxrandr-dev
+    libgirepository1.0-dev
+  DEB_BUILD_DEPS: |
+    ccache
+    autoconf-archive
+    autopoint
+    gir1.2-freedesktop
+    git
+    gobject-introspection
+    gtk-doc-tools
+    lsb-release
+    make
+    mate-common
+    meson
+    yelp-tools
+    iso-codes
+    gobject-introspection
+  # TODO
+  DEB_SCAN_BUILD_DEPS: |
+    clang
+    clang-tools
```

Comment:
Is scan-build really used?
I don't see it in build logs

Reply:
Ok, is see there is a ToDO :)
Can you mark the dependencies which are needed to build mate-desktop package?
Same what i did in travis config. https://github.com/mate-desktop/mate-panel/blob/master/.build.yml#L25

Reply:
Personally i read only build logs with their warnings in CI, so i can live pretty well without scan-build.
Of course it's enough to use it for one docker container if we want to use it.